packages/flutter/test/material/tabs_test.dart

```diff
@@ -4323,6 +4322,71 @@ void main() {
     expect(pageController.page, 2);
   });
 
+  testWidgets('Do not crash when the new TabController.index is longer than the old length.', (WidgetTester tester) async {
+    // This is a regression test for https://github.com/flutter/flutter/issues/97441
+
+    Widget buildFrame(TabController controller, bool showLast) {
```

Comment:
nit: make `showLast` a (required) named parameter. This makes the code below a little more readable as you can see what parameter is set to true/false.